### PR TITLE
Migrate PyTorch ResNet50 benchmarks

### PR DESCRIPTION
**Pull request by pzread**
Migrate PyTorch ResNet50 benchmarks from iree-samples.

This is based on #58

### CODE REVIEW

File: comparative_benchmark/pt_inductor/benchmark_all.sh [hunk 1]
```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+#
+# Copyright 2023 The OpenXLA Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
```
Comment (mariecwhite) [inline comparative_benchmark/pt_inductor/benchmark_all.sh:7]: nit: Add comments about example usage and supported environment variables

File: comparative_benchmark/pt_inductor/run_benchmarks.py [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+  parser = argparse.ArgumentParser(
+      description="Run PyTorch benchmarks with Inductor backend.")
+  benchmark_lib.configure_parser(parser)
+  return parser.parse_args()
+
+
+def main(**kwargs):
+  benchmark_lib.benchmark(benchmark_function=_run_framework_benchmark,
+                          benchmark_cases=benchmark_definitions.ALL_BENCHMARKS,
+                          compiler="pt-inductor",
```
Comment (mariecwhite) [inline comparative_benchmark/pt_inductor/run_benchmarks.py:170]:
```suggestion
                          compiler="inductor",
```

**Review by mariecwhite — APPROVED** [review]

File: comparative_benchmark/pt_inductor/benchmark_all.sh (hunk 1 above)
Comment (pzread) [reply comparative_benchmark/pt_inductor/benchmark_all.sh:7]: Done.

**Comment by pzread** [comment]
ResNet50 large batch sizes also failed due to out of CUDA memeory. Documented here #61

**Pull request merged**